Lesson23/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Accordion Example</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="accordion">
+        <div class="header-main">
+            <h1>Ingredients for FrontEnd</h1>
+        </div>
+        <div class="accordion-item">
+            <button class="accordion-header" onclick="toggleAccordion(event)">HTML</button>
```

Comment:
Avoid using 'onclick' attribute in HTML directly. Instead, select element and add event listener for it in JS, this makes your code more flexible and easier to work with.